app/javascript/mastodon/features/status/index.js

```diff
@@ -65,10 +67,19 @@ export default class Status extends ImmutablePureComponent {
     intl: PropTypes.object.isRequired,
   };
 
+  constructor (props) {
+    super(props);
+    this.state = {};
```

Comment:
I thought it was better to set it to the initial value of `fullscreen` of state.